torch/csrc/jit/codegen/cuda/scheduler/pointwise_heuristic.h

```diff
@@ -9,7 +9,7 @@ namespace jit {
 namespace fuser {
 namespace cuda {
 
-// Parameters the Reduction Heuristic Generates to describe the optimial
+// Parameters the Pointwise Heuristic Generates to describe the optimial
```

Comment:
nit: can't parse this English. Just "Parameters of the pointwise heuristic to describe..."?

Reply:
Split out and handled in https://github.com/csarofeen/pytorch/pull/1866